zebROS_ws/src/behaviors/src/game_piece_path_gen.cpp

```diff
@@ -44,12 +76,33 @@ bool genPath(behavior_actions::GamePiecePickup::Request &req, behavior_actions::
 	const size_t objects_num = std::min((int)(lastObjectDetection.objects.size()), (int)(req.max_objects));
 	ROS_INFO_STREAM("game_piece_path_gen : objects_num: " << objects_num);
 
-	std::vector<std::array<double, 3>> points;
-	points.push_back({0, 0, 0});
+	geometry_msgs::TransformStamped robotToMapTransform = tfBuffer->lookupTransform("map", "base_link", ros::Time(0));
```

Comment:
Since the requested path is being generated using the frame id in the header of the lastObjectDetection message, I think that would be the correct source frame to use here instead of map.

Reply:
oh, thanks! I'll fix that during tonight's meeting